server.js

```diff
@@ -27,6 +17,123 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+
+//GETS THE WHOLE DATA-SET UNFILTERED
+app.get('/collection', (req, res) => {
+  res.json(netflixData)
+})
+
+
+//GETS ALL MOVIES
+app.get('/movies', (req, res) => {
+  const onlyMovies = netflixData.filter((item) => item.type === "Movie")
+
+  res.json(onlyMovies)
+})
+
+
+//LOOKS FOR ALL MOVIES FROM A SPECIFIC YEAR AND ADDITIONAL QUERY FOR COUNTRY
+app.get('/movies/:year', (req, res) => {
+  const year = req.params.year
+  const onlyMovies = netflixData.filter((item) => item.type === "Movie")
+  let moviesFromYear = onlyMovies.filter((item) => item.release_year === +year)
+  const country = req.query.country
+
+  if (country) {
+    console.log(country)
+    moviesFromYear = moviesFromYear.filter((item) => item.country.toLowerCase() === country.toLowerCase())
+  }
+
+  res.json(moviesFromYear)
+})
+
+
+// !!THIS IS SUPPOST TO FILTER TYPE OF NETFLIX CONTENT AND COUNTRY WITH QUERIES BUT IT'S NOT WORKING
+//AND I DON'T UNDERSTAND WHY...
+
+// app.get('/collection', (req, res) => {
+//   let collection = netflixData
+//   const typeOfContent = req.query.type
+//   const showCountry = req.query.country
+
+//   if(typeOfContent) {
+//     collection = collection.filter((item) => item.type.toLowerCase() === typeOfContent.toLowerCase())
+//   }if(showCountry){
+//     collection = collection.filter((item) => item.country.toLowerCase() === showCountry.toLowerCase())
+//   }
+//   res.json(collection)
+// })
```

Comment:
Did you have both this and this:
```
//GETS THE WHOLE DATA-SET UNFILTERED
app.get('/collection', (req, res) => {
  res.json(netflixData)
})
```
at the same time, I think that's the reason why. Just delete the other one and comment this back in. If the user doesn't write any queries it will return an unfiltered result 👍